frontends/web/src/routes/account/walletconnect/connect.module.css

```diff
@@ -19,6 +19,9 @@
 
 
 @media (max-width: 768px) { 
+    .contentContainer {
+        margin-top: var(--space-half);
+    }
```

Comment:
oups this is a leftover, removing connect.module.css as this PR refactors connect.tsx to not use this CSS module.